internal/server/data/migrations.go

```diff
@@ -655,3 +637,19 @@ func deleteDuplicates(tx *gorm.DB, group string, model models.Modelable) error {
 	subQuery := tx.Select("min(id)").Group(group).Model(model)
 	return tx.Where("id NOT in (?)", subQuery).Delete(model).Error
 }
+
+// nolint:revive
+func addFieldsFor_0_14() *migrator.Migration {
+	return &migrator.Migration{
+		ID: "2022-07-21T18:28 add fields from 0.14.0",
```

Comment:
I amended the commit to use a bit more verbose name for the migration IDs. I think this reads a lot better and should work just as well as the old IDs.